cmd/neofs-cli/internal/common/exit.go

```diff
@@ -37,6 +42,9 @@ func ExitOnErr(cmd *cobra.Command, errFmt string, err error) {
 	case errors.As(err, &accessErr):
 		code = aclDenied
 		err = fmt.Errorf("%w: %s", err, accessErr.Reason())
+	case errors.Is(err, ErrAwaitTimeout):
+		code = aclTimeout
+		err = fmt.Errorf("%w: %s", err, ErrAwaitTimeout)
```

Comment:
u dont need this, `err` already has everything u need, e.g. `await container to be created: await timeout expired`